helm/cleanup.sh

```diff
@@ -12,6 +12,5 @@ kubectl delete crd clusterimagepolicies.securityenforcement.admission.cloud.ibm.
 
 kubectl delete jobs -n ${NAMESPACE} create-admission-webhooks create-armada-image-policies create-crds validate-crd-creation --ignore-not-found=true
 
-helm delete --purge ${RELEASE_NAME}
+helm delete ${RELEASE_NAME} --no-hooks 
```

Comment:
Why do we need `--no-hooks`? Do we not need that to remove the CRDs?

Reply:
we have just deleted the crds with `kubectl` a few lines before, I assert that hooks which run in delete are now redundant since we have just removed the resources, alss as a recover from broken state script we want to avoid waiting for failing hooks to timeout